components/identifier-generator/back/src/Infrastructure/Controller/GetIdentifierAttributesController.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Akeneo\Pim\Automation\IdentifierGenerator\Infrastructure\Controller;
+
+use Akeneo\Pim\Structure\Component\Query\PublicApi\Attribute\FindFlattenAttributesInterface;
+use Akeneo\Pim\Structure\Component\Query\PublicApi\Attribute\FlattenAttribute;
+use Akeneo\Platform\Bundle\FrameworkBundle\Security\SecurityFacadeInterface;
+use Akeneo\UserManagement\Bundle\Context\UserContext;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\AccessDeniedHttpException;
+
+final class GetIdentifierAttributesController
+{
+    // The maximum count of identifier attributes in the PIM
+    const MAX_RESULTS = 1;
```

Comment:
If not meant to be accessed externally:

```suggestion
    private const MAX_RESULTS = 1;
```